src/main/java/it/pagopa/pm/gateway/controller/XPayPaymentController.java

```diff
@@ -72,6 +73,9 @@ public class XPayPaymentController {
     @Autowired
     private XPayUtils xPayUtils;
 
+    @Autowired
+    private JwtTokenUtils jwtTokenUtils;
+
```

Comment:
Prefer constructor injection to field injection

Reply:
Done https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/210/commits/346e0aed4f064be62656a816027f182d0bef5fdd